docs/examples/paddle/ssd/train.py

```diff
@@ -75,8 +75,10 @@ def __init__(self,
             resize_x=300,
             resize_y=300,
             min_filter=types.DALIInterpType.INTERP_TRIANGULAR)
-        self.bc = ops.BrightnessContrast(device="gpu")
-        self.hsv = ops.Hsv(device="gpu")
+        self.hsv = ops.Hsv(device="gpu", dtype=types.FLOAT)
```

Comment:
I would add some comment that we need data between ops.Hsv and ops.BrightnessContrast in float to preserve accuracy.